cumulusci/tasks/bulkdata/extract.py

```diff
@@ -108,7 +108,7 @@ def _soql_for_mapping(self, mapping):
         sf_object = mapping["sf_object"]
         fields = self._fields_for_mapping(mapping)
         soql = f"SELECT {', '.join(fields)} FROM {sf_object}"
-        if "record_type" in mapping:
+        if mapping.record_type:
             soql += f" WHERE RecordType.DeveloperName = '{mapping['record_type']}'"
```

Comment:
Need to use mapping.record_type inside the f-string too, no?

Reply:
The MappingStep object is largely backwards compatible with the dict but I didn't implement __contains__ behaviour. Nevertheless, code consistency is a good thing so I'll change that.